static/css/components/shareLinks.less

```diff
@@ -58,3 +58,160 @@
     color: @grey;
   }
 }
+
```

Comment:
```suggestion

.share-link-container a {
  display: flex;
  justify-content: space-between;
  align-items: center;
  line-height: normal;
  width: 100%;
  text-decoration: none;

  img {
    margin-right: 10px;
    cursor: pointer;
    height: 30px;
  }
}
```

Reply:
Additionally, the following rules should be added to `.shareLinks-list a` (selector found on line 37):
`width: 80%;`
`margin: 0 auto;`